### DISCUSSION TITLE
hi, its possible to use github.io subdomains for another host ?

### DISCUSSION THREAD

**Discussion by ezioadf2** [community/community#39576] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hi,
i want run wordpress for my project with github subodmain, for some reasson i want use github subdomain surly and i know git hub suggest to use its host services, if i want use github.io subdomin to another hosting service or server , can i do it?

**Comment by airtower-luna** [comment, marked as answer]
No, that domain is for GitHub Pages sites only.
